src/System.Runtime.Serialization.Formatters/tests/FormatterServicesTests.cs

```diff
@@ -29,21 +34,260 @@ public void GetSerializableMembers_Interface()
         }
 
         [Fact]
-        public void GetUninitializedObject_InvalidArguments_ThrowsException()
+        public void GetUninitializedObject_NullType_ThrowsArgumentNullException()
         {
             Assert.Throws<ArgumentNullException>("type", () => FormatterServices.GetUninitializedObject(null));
             Assert.Throws<ArgumentNullException>("type", () => FormatterServices.GetSafeUninitializedObject(null));
         }
 
+#if netcoreapp
```

Comment:
This is great to know! Looks like the default constructor for `Type` was added back recently. I've extracted this to a common helper, as I know I can update `System.Runtime.Tests` to use this for no runtime types everywhere